src/unit.js

```diff
@@ -0,0 +1,38 @@
+define([
+	"./core",
+	"./unit/format",
+
+  "./plural"
+], function( Globalize, unitFormat ) {
+
+/**
+ * Globalize.formatUnit( value, unit, options )
+ *
+ * @value [Number]
+ *
+ * @unit [String]:
+ *
+ * @options [Object]
+ * - form: [String] "long", "short" (default), or "narrow".
+ *
+ * Format units such as seconds, minutes, days, weeks, etc.
+ */
+Globalize.prototype.formatUnit = function( value, unit, options ) {
+  var pluralGenerator;
+
+	if ( typeof value !== "number" ) {
+		throw new Error( "Value is not a number" );
+	}
+
+	if ( !unit ) {
+		throw new Error( "Missing unit" );
+	}
+
+	pluralGenerator = this.pluralGenerator();
+
+	return unitFormat( value, unit, options, pluralGenerator, this.cldr, this );
+};
```

Comment:
Your gist diff seems perfect!

The point of keeping `src/unit/format.js` CLDR-free (and as light as possible) is that it goes in the runtime module. The other functions that setup and prepare/create the `properties` (e.g., `src/unit/get.js`) can be safely ignored in the runtime modules.